backend/main.py

```diff
@@ -11,5 +11,19 @@ async def root():
 async def get_boba_list():
   with open('./database/bobalist.json') as f:
       # Load the JSON data into a Python object
-      data = json.load(f)
-  return data
+    bobaList = json.load(f)
+  return bobaList["bobaList"]
+
+@app.get('/milk_option')
+async def get_milk_option():
+  with open('./database/milkoption.json') as f:
+    milkOption = json.load(f)
+  return milkOption["milkOption"]

```

Comment:
Update the key inside your json as data

bobaList["data"]
milkOption["data"]
Etc.

Right now it's looking redundant

Reply:
resolved in 98846c08edea61cd887cc514e00f1cf4d2ac55bf